src/themes/shared/colors.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import lightColorTokens from '../light-theme/color-tokens.js';
+import getLightSemanticColorTokens from '../light-theme/color-semantic-tokens.js';
+import getLightComponentColorTokens from '../light-theme/color-component-tokens.js';
+import getLightDeprecatedSemanticColorTokens from '../light-theme/color-deprecated-semantic-tokens.js';
+import darkColorTokens from '../dark-theme/color-tokens.js';
+import getDarkSemanticColorTokens from '../dark-theme/color-semantic-tokens.js';
+import getDarkComponentColorTokens from '../dark-theme/color-component-tokens.js';
+import getDarkDeprecatedSemanticColorTokens from '../dark-theme/color-deprecated-semantic-tokens.js';
```

Comment:
We should not combine dark and light themes. First, it's not scalable, second, it will increase the bundle size with a default theme.